addons/l10n_in_edi_ewaybill/tests/test_edi_ewaybill_json.py

```diff
@@ -113,3 +114,24 @@ def test_edi_json(self):
             "totInvValue": 0.0
         })
         self.assertDictEqual(json_value, expected, "Indian EDI with 0(zero) quantity sent json value is not matched")
+
+        #=================================== Empty HSN test =============================================
+
+        json_value = self.env["account.edi.format"]._l10n_in_edi_ewaybill_generate_json(self.invoice_empty_product_hsn)
+        expected.update({
+            "docNo": "INV/2019/00010",
+            "itemList": [{
+                "productName": "product_a", "hsnCode": "", "productDesc": "product_a", "quantity": 1.0,
+                "qtyUnit": "UNT", "taxableAmount": 0.0, "cgstRate": 0.0, "sgstRate": 0.0
+            }],
```

Comment:
Would be prettier and would make better diff is properly splitted on multiple lines, but it's consistent so it's up to you

Reply:
It's better to keep it as it is because, in the above tests, we use the same way so it's consistent.